<launch>

	<rosparam command="load" file="$(find gnc_control)/config/control.yaml"/>
	<rosparam command="load" file="$(find gnc_control)/config/encoder_settings.yaml"/>
	<!-- Launch hardware interface then add the controller manager and controllers-->
	<node name="robot_hardware_interface" 
		pkg="gnc_control" 
		type="robot_hardware_interface" 
		output="screen"/>
	<node name="controller_manager_node" 
		pkg="controller_manager"
		type="spawner"  
		output="screen"  
		ns="/gnc_robot" 
		args="gnc_robot_velocity_controller 
			gnc_robot_joint_publisher" />

</launch>